pygitguardian/config.py

```diff
@@ -4,3 +4,4 @@
 
 MULTI_DOCUMENT_LIMIT = 20
 DOCUMENT_SIZE_THRESHOLD_BYTES = 1048576  # 1MB
+GENERAL_MAXIMUM_PAYLOAD = 2621440  # 25MB
```

Comment:
The `general__` part of the field name is an implementation detail of the server. I'd rather hide it and name this constant `MAXIMUM_PAYLOAD_SIZE`.